Content.Shared/VendingMachines/VendingMachineInventoryPrototype.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Content.Shared.VendingMachines
 {
-    [Serializable, NetSerializable, Prototype("vendingMachineInventory")]
+    [Serializable, NetSerializable, Prototype("VendingMachineInventory")]
```

Comment:
I thought style guide was camelCase?

Reply:
No, https://hackmd.io/Awj5TvdXTaOmhU2XZOsdQg#Naming.

Reply:
See 
![image](https://user-images.githubusercontent.com/31366439/114891590-5a542100-9e4f-11eb-97cd-f5aac92cdaf4.png)

Reply:
You're correct, fixed.